File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var answer = 0;
    
    let Max1 = 1
    let Max2 = 1
    
    for(const [w, h] of sizes) {
        const [cMin, cMax] = w > h ? [h,w] : [w,h]
        if(Max1 < cMax) {
            Max1 = cMax
        }
        if(Max2 < cMin) {
            Max2 = cMin
        }
    }
    console.log(Max1, Max2)
    return Max1 * Max2;
}

File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    
    first = 0 
    second = 0
    for x, y in sizes:
        ma, mi = max([x,y]), min([x,y])
        if ma > first:
            first = ma
        if mi > second:
            second = mi
            
        # a_x.append(x)
        # a_y.append(y)
        
    
    return first * second